tests/dynamic_data/publications/map_layer_relation/patch_after_feature_change_test.py

```diff
@@ -19,6 +19,7 @@
 
 @pytest.mark.timeout(60)
 @pytest.mark.usefixtures('oauth2_provider_mock')
+@pytest.mark.xfail(reason='Map filesystem input_file in not yet ready for WMS layers named by UUID')
```

Comment:
The reasondoes not make sense to me in this test

Reply:
As Layman does not parse map json correctly and does not know layer-map relation, it does not know to refresh map after layer patch.